pkg/service/appvendor/handler.go

```diff
@@ -0,0 +1,141 @@
+package appvendor
+
+import (
+	"context"
+	"time"
+
+	"openpitrix.io/openpitrix/pkg/db"
+	"openpitrix.io/openpitrix/pkg/gerr"
+	"openpitrix.io/openpitrix/pkg/manager"
+
+	"openpitrix.io/openpitrix/pkg/logger"
+	"openpitrix.io/openpitrix/pkg/models"
+	"openpitrix.io/openpitrix/pkg/pb"
+	"openpitrix.io/openpitrix/pkg/util/pbutil"
+)
+
+type Handler struct {
+}
+
+func (h *Handler) DescribeVendorVerifyInfos(ctx context.Context, req *pb.DescribeVendorVerifyInfosRequest) (*pb.DescribeVendorVerifyInfosResponse, error) {
```

Comment:
Use `func (p *Server) ` here will be better, then no need to add `serverImpl.go`.